spec/models/user_spec.rb

```diff
@@ -44,4 +44,20 @@
       end
     end
   end
+
+  describe '#has_never_logged_in?' do
+    context 'user has never logged in' do
+      it 'returns true' do
+        user = User.new(logged_in_at: nil)
+        expect(user.has_never_logged_in?).to be true
```

Comment:
This could also be written as `expect(user).to have_never_logged_in` and is more readable imo

But the expectation in the other context would read `expect(user),not_to have_never_logged_in` which sounds complicated.

Reply:
Agree; I'm not a fan of the `have_*` syntax